notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -0,0 +1,1689 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    ! pip install pipelines/components/google-cloud/.
</pre><p>You can pip install directly from Github without cloning:</p><p> </p><p><code>pip install git+https://github.com/kubeflow/pipelines.git</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Those lines aren't equivalent. `pip install git+https://github.com/kubeflow/pipelines.git` fails because the setup.py file isn't at the root of that git repo. I also don't want to install all of the KFP repo, just the GCPC part.

Reply:
There seems to be a way to do with with only pip, but I think what is occurring is more clear with git clone.